### PR TITLE
[No QA] Configured Reanimated 2 for Drawer

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ #3972

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [x] Android

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "8.0.4",
     "react-native-plaid-link-sdk": "^7.0.5",
-    "react-native-reanimated": "^2.1.0",
+    "react-native-reanimated": "^2.3.0-alpha.1",
```
Comment (roryabraham) [inline package.json:89]: Did you just do `npm i react-native-reanimated@next` ? Latest release on GH looks like `2.2.0`.

Comment (parasharrajat) [reply package.json:89]: I picked the most stable from the website https://docs.swmansion.com/react-native-reanimated/docs

**Review by roryabraham — APPROVED** [review]
Cool, this looks good to me.

**Review by mountiny — APPROVED** [review]
LGTM!

**Comment by marcaaron** [comment]
Please do update pods and commit the result anytime a `package.json` version is changed

```diff
diff --git a/ios/Podfile.lock b/ios/Podfile.lock
index 05fe3d6ea..1e8aa5bea 100644
--- a/ios/Podfile.lock
+++ b/ios/Podfile.lock
@@ -461,7 +461,7 @@ PODS:
     - React-Core
   - RNPermissions (3.0.1):
     - React-Core
-  - RNReanimated (2.1.0):
+  - RNReanimated (2.3.0-alpha.1):
     - DoubleConversion
     - FBLazyVector
     - FBReactNativeSpec
```

this change should have been included here 🙃

**Comment by parasharrajat** [comment]
Oh sorry. Totally forgot that I upgraded to the version to pod needs to updated as well.

**Pull request merged**